### PR TITLE
Minor improvements to BlogMemberExtension

### PR DESCRIPTION
**Pull request by flashbackzoo**

### CODE REVIEW

File: code/extensions/BlogMemberExtension.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
 class BlogMemberExtension extends DataExtension {
 
 	private static $db = array(
-		'URLSegment' => 'Varchar',
+		'URLSegment' => 'Varchar(50)',
```
Comment (tractorcow) [inline code/extensions/BlogMemberExtension.php]: If a name is more than 50 characters, you will en up with a database error.

You'll need to do a substr to limit the max characters in your generateURLSegment :)

**Comment by dhensby** [comment]
Unique index on `URLSegment`?

Only generate a `URLSegment` if there isn't one already?

**Comment by assertchris** [comment]
@flashbackzoo want to pursue this, or should we close?

**Comment by flashbackzoo** [comment]
close

**Pull request closed without merging**